src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1585,7 +1585,8 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMValidation> const& m)
             return;
         }
 
-        bool isTrusted = app_.getUNL ().nodeInUNL (val->getSignerPublic ());
+        auto const isTrusted =
+            app_.validators().member (val->getSignerPublic ());
```

Comment:
The type if isTrusted is boost::optional<string>, but the variable name signals that it's of type bool (and that's how it's used below). We could cast the result of `member` to bool (or add a helper to to validators that returns a bool).